core/src/test/kotlin/com/malinskiy/marathon/cache/SimpleEntryReader.kt

```diff
@@ -0,0 +1,20 @@
+package com.malinskiy.marathon.cache
+
+import kotlinx.coroutines.io.ByteReadChannel
+import kotlinx.coroutines.io.readRemaining
+import kotlinx.io.charsets.Charset
+import kotlinx.io.core.readBytes
+
+class SimpleEntryReader : CacheEntryReader {
+
+    var readInvoked: Boolean = false
+    var bytes: ByteArray? = null
+    val data: String?
+        get() = bytes?.toString(charset = Charset.defaultCharset())
```

Comment:
I would better specify UTF-8 as StandardCharsets.UTF_8, because for Windows default charset in CP1251, and we shouldn't depend on OS

Reply:
ok, will change to UTF-8